flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/config/entries/DeploymentEntry.java

```diff
@@ -161,7 +165,11 @@ public static DeploymentEntry enrich(DeploymentEntry deployment, Map<String, Str
 		});
 
 		final DescriptorProperties properties = new DescriptorProperties(true);
-		properties.putProperties(enrichedProperties);
+		try {
```

Comment:
instead of throwing a `SqlExecutionException` in a configuration class, we should rather throw it in the `LocalExecutor`. All those methods already define a `throws SqlExecutionException`. This also allows to catch exceptions in both `DeploymentEntry` and `EnvironmentEntry` and `ConfigurationEntry` at the same time.